scripts/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
 
   mysql:
     image: mysql
-    command: --default-authentication-plugin=mysql_native_password
+    command: --default-authentication-plugin=mysql_native_password --sync_binlog=0 --skip-innodb_doublewrite --innodb-flush-log-at-trx-commit=0 --innodb-flush-method=nosync
```

Comment:
can you add next to this line comment, explaining rationale behind these parameters?

Reply:
Ok, later